plugins/node.d/http_loadtime.in

```diff
@@ -54,6 +60,19 @@ escapeUri() {
     echo "$1" | sed 's/[:/.-]/_/g'
 }
 
+request_url_with_args_from_env() {
+    uri=$1
```

Comment:
Please use `local` for function-specific variables (`uri` and `cookies_file`).

Reply:
Shellcheck in PyCharm does warn me that `local` is not POSIX: https://github.com/koalaman/shellcheck/wiki/SC3043

Will use the suggested prefix solution instead.

Reply:
Sadly `shellcheck` assumes the traditional POSIX shell instead of its most common real incarnations (`dash` or similar).
Our linting uses `shellcheck -s dash` for all files with `#!/bin/sh`. This should be your reference.